tutorials/2d/2d_lights_and_shadows.rst

```diff
@@ -0,0 +1,226 @@
+color in the top section. By default shadows are turned off and the ``mode`` is set to ``add``. This 
+means that each light adds its own color to whatever is underneath.
+
+.. image:: img/light_shadow_all_lights_no_blob.png
+
+Next child a :ref:`Sprite <class_Sprite>` to each of the :ref:`Light <class_Light2D>` nodes, and set 
+the :ref:`Sprite's <class_Sprite>` texture to the `blob image <https://raw.githubusercontent.com/godotengine/godot-demo-projects/master/2d/lights_and_shadows/spot.png>`_. Each of these 
+should stay centered on the :ref:`Light <class_Light2D>` node. The blob is the image of the light 
+itself while the :ref:`Light <class_Light2D>` shows the effect that the light has on the scene. The 
+:ref:`LightOccluder2D's <class_LightOccluder2D>` will treat the position of the light as the center of the :ref:`Light <class_Light2D>` 
+node which is why we want the blob to be centered on its parent :ref:`Light <class_Light2D>`.
+
+.. image:: img/light_shadow_all_lights.png
+
+.. note:: At the time of writing this tutorial the stable version is 3.0 but the master version that will become 3.1
+          contains many changes to the animation system. So the animations contained in the demo will not be covered.
+
+Right now the scene should look too bright this is because all three lights are adding color to the scene. 
+This is why the demo uses a :ref:`CanvasModulate <class_CanvasModulate>` in the scene. The
+:ref:`CanvasModulate <class_CanvasModulate>` multiples the entire viewport by a specific color.
+
+Add a :ref:`CanvasModulate <class_CanvasModulate>` to the scene and set its color to ``rgb(70, 70, 70)``. 
+This will make the scene sufficiently dark to see the effects of the lights distinctly.
+
+.. image:: img/light_shadow_ambient.png
+
+Now we add the shadow casters.
+
+The demo uses a :ref:`Node <class_Node2D>` named "casters" to organize the shadow casters. Add a 
+:ref:`Node2D <class_Node2D>` to the scene. It will be used to group all the shadow casters together. 
+This way we can show and hide them all at the same time.
+
+Each shadow caster is made of a :ref:`Sprite <class_Sprite>`, with a :ref:`LightOccluder2D <class_LightOccluder2D>` 
+child. For the demo the :ref:`Sprite <class_Sprite>` has a texture 
+set to the `caster image <https://raw.githubusercontent.com/godotengine/godot-demo-projects/master/2d/lights_and_shadows/caster.png>`_ and nothing else. The child :ref:`LightOccluder2D <class_LightOccluder2D>` is where all the magic happens. In a 
+game the :ref:`Sprite <class_Sprite>` could be more than a black box, it could be an image of whatever object is casting 
+the shadow, it could be a wall, a magical chest, or anything else.
+
+.. image:: img/light_shadow_sprites.png
+
+:ref:`LightOccluder2Ds <class_LightOccluder2D>` tell the game what the occluder looks like. They hold 
```

Comment:
Better to say they tell what *shape* the occluder has. As specified in the previous paragraph, a Sprite is used to give an appearance.